ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/CustomExecutors.java

```diff
@@ -116,8 +117,9 @@ private static StatementExecutor insertValuesExecutor() {
     final InsertValuesExecutor executor = new InsertValuesExecutor();
 
     return (statement, executionContext, serviceContext) -> {
-      executor.execute(statement, executionContext, serviceContext);
-      return Optional.empty();
+      final Optional<String> message =
+          executor.execute(statement, executionContext, serviceContext);
+      return Optional.of(new MessageEntity(statement.getStatementText(), null, message));
```

Comment:
```suggestion
      return executor.execute(statement, executionContext, serviceContext)
           .map(msg -> new MessageEntity(statement.getStatementText(), msg));
```

Reply:
ok

Reply:
Unfortunately the suggestion doesn't compile as is. 
I think would need to:
```
return executor.execute(statement, executionContext, serviceContext)
 .map(msg -> new MessageEntity(statement.getStatementText(), (String)msg));
```

To do it on one line? Which seems a bit ugly.

Might be easier and clearer to split across two lines:

```
final Optional<String> res = executor.execute(statement, executionContext, serviceContext);
      return res.map(msg -> new MessageEntity(statement.getStatementText(),
              null, Optional.of(msg)));
```

Reply:
Why would you need to cast `msg` to `String`? It is a `String`!  

It's probably not compiling for you at the moment because `MessageEntity` takes an `Optional<String>`, (which, as per above, I don't think it should).